nixos/lib/utils.nix

```
@@ -343,6 +343,25 @@ utils = rec {
     in
       filter (x: !(elem (getName x) namesToRemove)) packages;
 
+  /* Returns false if `package` with the same name is present in `packagesToDisable`.
+
+     Type:
+       enableByPackageName :: package -> [package] -> bool
+
+     Example:
+       enableByPackageName file-roller [ file-roller totem ]
+       => false
+
+     Example:
+       enableByPackageName nautilus [ file-roller totem ]
+       => true
+  */
+  enableByPackageName = package: packagesToDisable:
+    let
+      namesToDisable = map getName packagesToDisable;
+    in
+      elem (getName package) namesToDisable;
```

Ugh, should be:

```suggestion
      !elem (getName package) namesToDisable;
```

I really need find time to test it properly.